test/functional/apps/management/_index_pattern_create_delete.js

```diff
@@ -28,12 +28,11 @@ export default function ({ getService, getPageObjects }) {
           .then(id => indexPatternId = id);
       });
 
-      it('should have index pattern in page header', function () {
-        return PageObjects.settings.getIndexPageHeading().getVisibleText()
-        .then(function (patternName) {
-          screenshots.take('Settings-indices-new-index-pattern');
-          expect(patternName).to.be('logstash-*');
-        });
+      it('should have index pattern in page header', async function () {
+        const indexPageHeading = await PageObjects.settings.getIndexPageHeading();
+        const patternName = await indexPageHeading.getVisibleText();
```

Comment:
Like `return`, `await` affects the entire *expression*, not just a function invocation.  So `await` doesn't really "kick in" until it finds a newline, a semi-colon, the closing of a block, etc.